doc/getting-started/connect-widget.rst

```diff
@@ -5,10 +5,9 @@ The easiest option for letting people connect their storage to your app is using
 the Connect Widget add-on library, which is written and maintained by the rs.js
 core team.
 
-This is by no means required, and it's easy to integrate all functionality in
-your own UI as well. However, it's a great way for starting out with RS app
-development and, if desired, replacing the widget with your own custom code
-later on.
+This is optional and an easy way to integrate all functionality into your own UI.
+It's a great way to start with RS app development and, can be replaced with
+custom code later on.
```

Comment:
This changes the meaning of the paragraph entirely. The intention here was to explain that widget is optional, and it's easy to NOT use it, but instead integrate the pure functionality in your own custom UI (not meaning to integrate the widget's UI in your UI).